.env.local

```diff
@@ -5,4 +5,5 @@ POSTGRES_PORT=5432
 POSTGRES_DB=waect
 SECRET_KEY=waectsk
 DATABASE_URL=postgresql://$POSTGRES_USER:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DB
+DATABASE_URL_PROD=
```

Comment:
I would put this in another file called .env.prod and assign its content to the same DATABASE_URL. (but for production)